src/main/java/frc/robot/subsystems/climb/lifter/Lifter.java

```diff
@@ -60,6 +74,9 @@ public void resetPosition(Rotation2d position) {
 
 	private void updateInputs() {
 		motor.updateInputs(positionSignal);
+
+		limitSwitch.updateInputs(limitSwitchInputs);
+		Logger.processInputs(getLogPath() + "/LimitSwitch", limitSwitchInputs);
```

Comment:
```suggestion
		Logger.processInputs(getLogPath() + "/LimitSwitch", limitSwitchInputs.getDebouncedValue());
```
או איך שלא קוראים לזה